packages/eds-core/src/utils/decorateBlocks.mts

```diff
@@ -5,5 +5,7 @@ import { decorateBlock } from './decorateBlock.mjs';
  * @param {Element} main The container element
  */
 export function decorateBlocks(main: Element) {
-	main.querySelectorAll('div.section > div > div').forEach(decorateBlock);
+	main
+		.querySelectorAll<HTMLElement>('div.section > div > div')
```

Comment:
This should be an `HTMLDivElement` because the last element has to be a div.
```suggestion
		.querySelectorAll<HTMLDivElement>('div.section > div > div')
```